test/unit/checkpoint/checkpoint.test.js

```diff
@@ -0,0 +1,5 @@
+describe('Checkpoint', () => {
+  it('should work', () => {
+    expect(true).toBe(true);
```

Comment:
I left this for now, as testing it would require more fixtures and that can be done at a later time.